src/couch/src/couch_httpd_db.erl

```diff
@@ -685,6 +685,7 @@ db_doc_req(#httpd{method = 'COPY'} = Req, Db, SourceDocId) ->
         end,
     {TargetDocId0, TargetRevs} = parse_copy_destination_header(Req),
     TargetDocId = list_to_binary(mochiweb_util:unquote(TargetDocId0)),
+    couch_db:validate_docid(Db, TargetDocId),
```

Comment:
what's the difference between this and the similar code above (`chttpd_db.erl`)?

Reply:
that's for the legacy 5986 node-local interface, just here for completeness.